Sources/PrimerSDK/Classes/PCI/Services/Network/URLSessionStack.swift

```diff
@@ -98,19 +104,19 @@ internal class URLSessionStack: NetworkService, LogReporter {
                     errorBody: nil,
                     responseCode: (response as? HTTPURLResponse)?.statusCode
                 )
+                resEventProperties = resEvent?.properties as? NetworkCallEventProperties
 
-                resEvent!.properties = resEventProperties
             }
 
 #if DEBUG
 
 #endif
 
             if let error = error {
-                if resEvent != nil {
-                    resEventProperties!.errorBody = "\(error)"
-                    resEvent!.properties = resEventProperties
-                    Analytics.Service.record(event: resEvent!)
+                if var resEvent = resEvent, var resEventProperties = resEventProperties {
+                    resEventProperties.errorBody = "\(error)"
+                    resEvent.properties = resEventProperties
+                    Analytics.Service.record(event: resEvent)
```

Comment:
this is solving a crash I just found during the testing of my NOL changes :) great!

Reply:
haha I suspected I wasn't the only one - this one is a real pain >_<